docs/index.rst

```diff
@@ -4,4 +4,315 @@ JDBC Connector
 ==============
 
 The JDBC connector allows you to import data from any relational database with a
-JDBC driver into Kafka topics.
+JDBC driver into Kafka topics. By using JDBC, this connector can support a wide variety of
+databases without requiring custom code for each one.
+
+Data is loaded by periodically executing a SQL query and creating an output record for each row
+in the result set. By default, all tables in a database are copied, each to its own output topic.
+The database is monitored for new or deleted tables and adapts automatically. When copying data
+from a table, the connector can load only new or modified rows by specifying which columns should
+be used to detect new or modified data.
+
+Quickstart
+----------
+
+To see the basic functionality of the connector, we'll copy a single table from a local SQLite
+database. In this simple example, we'll assume each entry in the table is assigned a unique ID
+and is not modified after creation. Start by creating a database (you'll need to install SQLite
+if you haven't already)::
+
+   $ sqlite3 test.db
```

Comment:
We are using SQLite for quickstart because thats the driver we can bundle? 
Because in terms of "quickstart" this is a database our users are least likely to be familiar with.